tests/requires/test.yaml

```diff
@@ -0,0 +1,40 @@
+requires:
+  files:
+    - rust/src/detect/requires.rs
+
+pcap: ../eve-metadata/testmyids.pcap
+
+checks:
+
+  # Check that for Suricata >= 8.0.0 we have one alert and that alert is sid 8.
+  - filter:
+      requires:
+        min-version: 8
+      count: 1
+      match:
+        event_type: alert
+  - filter:
+      requires:
+        min-version: 8
+      count: 1
+      match:
+        alert.signature_id: 8
```

Comment:
These could be one filter. Same for below filters. Why were they written separately?

Reply:
I don't think so. We want to confirm 2 things:
- There is only 1 alert, regardless of SID
- That alert is SID 8

If these checks were to be combined, we'd be testing that there is 1 or more alerts, but only one has SID 8. But there could still be more alerts with other SIDs.

Reply:
ah. ok. thanks for explaining!